cinema/models.py

```diff
@@ -5,6 +5,32 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField("Genre", blank=True, related_name="genres")
```

Comment:
The `related_name` for the `genres` field in the `Movie` model is set to 'genres', which is not unique or descriptive. Consider using a more descriptive name like 'movies_as_genre' to indicate the reverse relationship.